src/components/ValidateCode/ValidateCodeModal.js

```diff
@@ -40,6 +40,8 @@ const defaultProps = {
 };
 
 function ValidateCodeModal(props) {
+    const signInHere = useCallback(() => Session.signInWithValidateCode(props.accountID, props.code), [props.accountID, props.code]);
```

Comment:
NAB - I hate this non-destructuring haha. Trying to figure out where we landed in [that discussion](https://expensify.slack.com/archives/C01GTK53T8Q/p1681915150185209?thread_ts=1680718704.040129&cid=C01GTK53T8Q) but this is fine for now even though ugly

Reply:
Update - apparently we are allowed to destructure - I think that might be a bit cleaner here!

Reply:
Okay I lied, we're not. Sigh. I'll approve this!